scripts/devel/create_devel_task_generator.py

```diff
@@ -71,6 +71,11 @@ def main(argv=sys.argv[1:]):
         action='store_true',
         help='The flag if the workspace should be built with tests enabled '
              'and instead of installing the tests are ran')
+    parser.add_argument(
+        '--run-abichecker',
+        dest='abichecking',
```

Comment:
Nitpick: please remove the custom `abichecking`. The default destination `run_abichecker` is just fine.

Reply:
7d60961